From 76a735064a3cf6130a43ba521ffba60c067229c3 Mon Sep 17 00:00:00 2001
From: Vasyl Gello <vasek.gello@gmail.com>
Date: Tue, 3 Mar 2020 12:37:19 +0000
Subject: [PATCH] Bump Security String to 2020-03-05

Implemented:
============
CVE:            References:     Type:   Severity:       Updated AOSP versions:
CVE-2020-0032	A-145364230	RCE	Critical	8.0, 8.1, 9, 10
CVE-2020-0034	A-62458770	ID	High		8.0, 8.1
CVE-2020-0036	A-144679405	EoP	High		8.0, 8.1, 9, 10
CVE-2020-0035	A-140622024	ID	High		8.0, 8.1, 9
CVE-2020-0038	A-143109193	ID	High		8.0, 8.1, 9, 10
CVE-2020-0039	A-143155861	ID	High		8.0, 8.1, 9, 10
CVE-2020-0037	A-143106535	ID	High		8.0, 8.1, 9, 10

Not Implemented:
================
CVE:            References:     Type:   Severity:       Updated AOSP versions:
None

Not Applicable (platform source):
=================================
CVE:            References:     Type:   Severity:       Updated AOSP versions:
CVE-2019-2194	A-137284057	EoP	High		9
CVE-2020-0029	A-140065828	ID	High		10
CVE-2020-0031	A-141703197	ID	High		10
CVE-2020-0033	A-144351324	EoP	High		8.0, 8.1, 9, 10

Signed-off-by: Vasyl Gello <vasek.gello@gmail.com>
Change-Id: I49ff32447357fe01d3d51b894d9c545abeb3a082
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 4c8a88f9c5..d30a9f0f0c 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -131,7 +131,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2020-02-05
+    PLATFORM_SECURITY_PATCH := 2020-03-05
 endif
 
 ifeq "" "$(PLATFORM_BASE_OS)"
-- 
2.17.1

